### PR TITLE
925953 - [dev] WinUI kanban control content changes added

### PR DESCRIPTION
**Pull request by Kerubananthan3761**
## Description
WinUI kanban control content changes added and API links added.

### CODE REVIEW

File: winui/Kanban/getting-started.md [hunk 1]
```diff
@@ -233,14 +236,136 @@ this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Code Review", Categor
...
 
 Refer to the following link for guidance on applying themes: [Themes for Syncfusion WinUI controls](https://help.syncfusion.com/winui/common/themes)
 
-N> You can refer to our `WinUI Kanban` feature tour page for its groundbreaking feature representations. You can also explore our `WinUI Kanban Examples` that shows you how to render the Kanban in WinUI.
\ No newline at end of file
+N> You can refer to our `WinUI Kanban` feature tour page for its groundbreaking feature representations. You can also explore our `WinUI Kanban Examples` that shows you how to render the Kanban in WinUI.
+
+## Localization
+
+The WinUI Kanban allows localization of all the static default strings, enabling you to translate them into any supported language. For detailed steps on how to localize the default strings, refer to the following link: [Localization in WinUI controls](https://help.syncfusion.com/winui/common/localization).
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:367]: The WinUI Kanban control supports the localization of all static default strings, allowing you to translate them into any supported language. For detailed instructions on how to localize the default strings, refer to the following link: [Localization in WinUI controls](https://help.syncfusion.com/winui/common/localization)

File: winui/Kanban/getting-started.md [hunk 2]
```diff
@@ -233,14 +236,136 @@ this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Code Review", Categor
...
+
+## Right to Left (RTL)
+
+The WinUI Kanban supports RTL rendering, displaying text and the control’s layout from right to left. For more information on enabling RTL rendering refer to the following link: [Right to left in WinUI controls](https://help.syncfusion.com/winui/common/right-to-left).
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:371]: The WinUI Kanban control supports RTL (Right-to-Left) rendering, allowing both text and the control's layout to be displayed from right to left. For more information on how to enable RTL rendering, please refer to the following link: [Right to left in WinUI controls](https://help.syncfusion.com/winui/common/right-to-left).

Comment (TamilarasanGunasekaran) [reply winui/Kanban/getting-started.md]: Changed.

File: winui/Kanban/getting-started.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/getting-started.md]: Changed.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**